board/docker-compose.yml

```
@@ -1,6 +1,6 @@
 services:
   board:
-    image: k8s-study/board:1.1
+    image: k8s-study/board:1.0
```

해당 파일은 이번 과제 내용에 포함되지 않는 파일입니다. 기존 설정으로 올려주세요

```suggestion
    image: k8s-study/board:1.1
```